tests/scripts/pulpcore/test_content_guards.sh

```diff
@@ -20,7 +20,9 @@ expect_succ pulp content-guard rbac show --name "cli_test_guard"
 
 expect_succ pulp group create --name "cli_test_group"
 expect_succ pulp content-guard rbac assign --name "cli_test_guard" --group "cli_test_group"
+expect_succ pulp content-guard rbac show --name "cli_test_guard"
 test "$(echo "$OUTPUT" | jq -r '.groups' | jq -r length)" -eq "1"
 expect_succ pulp content-guard rbac remove --name "cli_test_guard" --user "admin" --group "cli_test_group"
+expect_succ pulp content-guard rbac show --name "cli_test_guard"
```

Comment:
The groups and users listed here are determined by the permission, right? This is a little bit dangerous, if the admin changes the access policy to use a different permission, or a non trivial condition. Also not a problem to solve here.

Reply:
Yes they are, something to think about.